Tests/Pcap++Test/Tests/IpMacTests.cpp

```diff
@@ -455,29 +460,29 @@ PTF_TEST_CASE(TestIPv4Network)
 PTF_TEST_CASE(TestIPv6Network)
 {
 	// Invalid c'tor: IPv6 address + prefix len
-	PTF_ASSERT_RAISES(pcpp::IPv6Network(pcpp::IPv6Address("invalid"), 1), std::invalid_argument, "address is not a valid IPv6 address");
-	PTF_ASSERT_RAISES(pcpp::IPv6Network(pcpp::IPv6Address("2001:db8::"), 129), std::invalid_argument, "prefixLen must be an integer between 0 and 128");
+	PTF_ASSERT_RAISE_INCLUDES(pcpp::IPv6Network(pcpp::IPv6Address("invalid"), 1), std::invalid_argument, "Not a valid IPv6 address");
```

Comment:
This test doesn't make sense anymore because the IPv6 addresss isn't valid. We can remove it

Reply:
fixed, and removed all similar cases.